01LessonOne/LessonOne.js

```diff
@@ -2,36 +2,62 @@ module.exports = function() {
 
   this.returnTrue = function () {
     // should return true
-
+	  return true;
+      
   }
 
   this.returnFalse = function() {
     // should return false
+	  return false;
 
   }
 
   this.greaterThan = function (big, small) {
     // should return true if the first argument is greater than the second argument
     // and return false if the first argument is less than the second argument
+	  if (big > small) {
```

Comment:
What kind of value does the expression big > small result in? What are the possible values? What are the circumstances that lead to the expression being a particular value?